src/main/java/com/google/api/codegen/nodejs/NodeJSGapicContextUtil.java

```diff
@@ -0,0 +1,28 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.nodejs;
+
+/**
+ * Utility methods for NodeJS GAPIC contexts.
+ */
+public class NodeJSGapicContextUtil {
```

Comment:
I don't think this functionality merits its own class. In general, I'd like to avoid an explosion of classes.